projects/js-packages/components/components/diff-viewer/README.md

```diff
@@ -1,6 +1,6 @@
 # Unified Diff Viewer
 
-Forked over from [Calypso](https://github.com/Automattic/wp-calypso/tree/b7a4a07/client/components/diff-viewer).
+Originally forked over from [Calypso](https://github.com/Automattic/wp-calypso/tree/b7a4a07/client/components/diff-viewer).
```

Comment:
If this is a copy of Calypso component, why aren't we consuming this via [`@automattic/components` package](https://github.com/Automattic/wp-calypso/tree/581f45d72e1f8f47bbb2e46956b13c18aeb6f7e2/packages/components#readme)?